File: Trabalho_Teorico_1/Unidade_00g/exercicio_13.java
/**
 * Marcos Antonio Lommez Candido Ribeiro
 * 77157
 * AED's 2
 * 13/08/2022
 */
class exercicio_13 {
    /*
     * Faca um programa que mostre os 10 primeiros numeros inteiros positivos.
     */
    public static void main(String[] args) {
        MyIO.print("Os 10 primeiros numeros positivos sao:\n");
        int i = 1;
        while (i<=10) {
            MyIO.print(i);
            MyIO.print(" ");
            i++;
        }
    }
}

File: Trabalho_Teorico_1/Unidade_00m/exercicio_13.c
/**
 * Marcos Antonio Lommez Candido Ribeiro
 * 77157
 * AED's 2
 * 14/08/2022
 */

#include <stdlib.h>
#include <stdio.h>
#include <math.h>

/**
 * Faca um programa que leia n numeros inteiros, armazene-os em um
 * arquivo, leia-os do arquivo e mostre-os na tela
 */
int main()
{
    FILE *file = fopen("exemplo.txt", "r");

    if (!file)
    {
        printf("exemplo.txt nao encontrado! saindo");
    }
    else
    {
        char tmp = '\0';
        fseek(file, 0L, SEEK_END);
        int size = ftell(file);
        for (int i = size-1; i >= 0; i--) // size-1 ignora o FEOF no fim do arquivo
        {
            fseek(file, i, SEEK_SET);
            fread(&tmp, sizeof(char), 1, file);
            printf("%c", tmp);
        }
    }
    fclose(file);

    return 0;
}